### PR TITLE
Auditbeat: Add placeholder for new docs directory and file

### PR DESCRIPTION
**Pull request by karenzone**
We need a placeholder for accurate testing and to break docs-ci gridlock between repos

* [Doc: Add docs for Auditbeat add_session_metadata processor #40252](https://github.com/elastic/beats/pull/40252) won't pass docs-ci because doc build is not pulling from the new repo. 
* [Add x-pack-auditbeat to conf.yaml docs#3038](https://github.com/elastic/docs/pull/3038) won't pass docs-ci because the files we're trying to pull in haven't been merged.

### CODE REVIEW

**Comment by karenzone** [comment]
@mjwolf will you please review/approve so that I can merge https://github.com/elastic/docs/pull/3038.  That PR is required to break the gridlock so that I can get accurate docs-ci checking on https://github.com/elastic/beats/pull/40252.

**Comment by karenzone** [comment]
Need code owner review to unblock this work, please.  (See description for details.)
Let me know if you have concerns.

**Review by mjwolf — APPROVED** [review]

**Comment by karenzone** [comment]
11816/7

**Pull request merged**
